orchagent/port.h

```diff
@@ -28,6 +28,9 @@ extern "C" {
 
 #define VNID_NONE               0xFFFFFFFF
 
+// Default inter-packet gap(IPG) is 96 which is recommended by IEEE 802.3
```

Comment:
Recommend adding 'bits' after 96 in comment line just to be explicit, clear

Reply:
Sure, Add it.